### ISSUE TITLE
Regarding the availability of 28-day mortality in MIMIC-IV V2.0

### ISSUE THREAD

**Issue by ningyile** [MIT-LCP/mimic-code#1411]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues/1199

### Description
MIMIC-IV V2.0 was released on June 12, 2022. The date of death is available within the 'dod' column of the 'patients' table since V2.0, and the maximum follow-up time for each patient is exactly one year after their last hospital discharge. With out-of-hospital follow-up mortality([ stop getting death from Social Security Death Index, but purchasing the data from the CDC](https://github.com/MIT-LCP/mimic-code/issues/1199)), MIMIC-IV V2.0 provides a basis for calculating 28-day mortality(same as 30-day, 60-day, 90-day, and even up to 1-year mortality, etc.).  Some scholars have used 28-day mortality as a clinical outcome in studies with MIMIC-IV V2.0.

[Zhu, Z., Zhou, M., Wei, Y. et al. Time-varying intensity of oxygen exposure is associated with mortality in critically ill patients with mechanical ventilation. Crit Care 26, 239 (2022)](https://ccforum.biomedcentral.com/articles/10.1186/s13054-022-04114-w). https://doi.org/10.1186/s13054-022-04114-w

However, I noticed that the source of out-of-hospital mortality was from '[state records](https://physionet.org/content/mimiciv/2.0/)’, and I‘m confused with the data source(I'm an ICU physician in China mainland, I am not familiar with the US government institutions), I mean whether the data was from the state CDC(Massachusetts) or the national CDC？

If the survival data come from the CDC of Massachusetts, for after discharge of Non-Massachusetts patients(who come from other states and even from other countries), their follow-up information can be tracked and recorded by the CDC of Massachusetts? I know the vast majority of patients in MIMIC-IV should be local residents, and only a very small percentage of patients may fall into the category I mentioned above. If out-of-hospital survival data for non-Massachusetts patients could not be tracked, could I exclude this part of non-Massachusetts patients based on certain fields(for example, insurance from admissions or other fields)?

As it's important for the clinical outcome of our project, I am interested in whether metrics such as 28-day mortality can be used in MIMIC-IV V2.0. In other words, I would like to ask whether the state(Massachusetts?) CDC data cover the vast majority of patients for out-of-hospital survival information enough. If there are only a few are not covered, the impact of those few can be ignored? 

In summary, 28-day mortality as the clinical outcome can be used in MIMIC-IV V2.0? 

@alistairewj

**Comment by shaou77** [comment]
Hi, I am also a Chinese ICU doctor. A word about something unrelated to your question. I read an article using 28-day mortality before MIMIC IV 2.0 was released, isn't that nonsense?

**Comment by ningyile** [comment]
> Hi, I am also a Chinese ICU doctor. A word about something unrelated to your question. I read an article using 28-day mortality before MIMIC IV 2.0 was released, isn't that nonsense?

You may have gotten an idea of the mortality of MIMIC-IV from [an issue proposed previously in the community](https://github.com/MIT-LCP/mimic-code/issues/1199). https://github.com/MIT-LCP/mimic-code/issues/1199. In fact, MIMIC-IV V1.0 only provided in-hospital death records, so any study with MIMIC-IV V1.0(or previous version) involving 28-day, 30-day, or 60-day mortality had the wrong clinical outcome. Unfortunately, some journal editors and reviewers ignored the problem. The good news is that the `dod` field in the `patients` table in MIMIC-IV V2.0 contains '[state records](https://physionet.org/content/mimiciv/2.0/)’ of out-of-hospital death which was purchased by the administrator from the CDC for over $70,000, with a maximum follow-up of 1 year after their last hospital discharge. This is meant to provide a basis for calculating the 28-day mortality, etc. As out-of-hospital death data comes from state records, not like the MIMIC III from Social Security Death Index. I am concerned that there may be a subset of non-local patients (compared to Massachusetts patients) that may not be covered. So I'm here to ask the official administrator.
